### ISSUE TITLE
`complete-docs` on type member

### ISSUE THREAD

**Issue by jdmichaud** [palantir/tslint#4396]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 3.2.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
class Foo {
  private someField?: { bar(): void };
}
```

with `tslint.json` configuration:

```json
    "completed-docs": [
      true,
      {
        "classes": true,
        "enums": true,
        "functions": {
          "visibilities": [
            "exported"
          ]
        },
        "methods": {
          "privacies": [
            "public"
          ]
        },
        "interfaces": true,
        "namespaces": true
      }
    ],
```

#### Actual behavior
```
Documentation must exist for methods.
```
on the `bar` function.
#### Expected behavior
The type is not an interface but a type. I don't expect tslint to complain about it.

**Comment by JoshuaKGoldberg** [comment]
@buu700 mentioned in #4398:

> Edit: Adding `"locations": "all"` to `"methods"` and `"properties"` fixed this. Is this expected?

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is deprecated** and no longer accepting pull requests other than security fixes. See #4534. ☠️ 
We recommend you instead use **[typescript-eslint](https://typescript-eslint.io)** to lint your TypeScript code with **ESLint**. ✅

👋 It was a pleasure open sourcing with you!

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
